cmd/cmd.go

```diff
@@ -494,7 +496,10 @@ func (o *options) generateTemplate() (*template.Template, error) {
 				levelColor = color.New(color.FgRed)
 			case "fatal":
 				levelColor = color.New(color.FgCyan)
+			case "critical":
+				levelColor = color.New(color.FgCyan)
 			default:
+				levelColor = color.New(color.FgWhite)
```

Comment:
```suggestion
				return level
```

Since some terminals have a default background color of white, maybe we should avoid using white. How about returning text with the default color?

Reply:
Yeah, that's a great idea!